### PR TITLE
Fixes for some crashes in Linux

### PR DESCRIPTION
**Pull request by xrombik**
Added fixes intended to find reasons for Linux crashes. Added some help for launch current build.

### CODE REVIEW

File: src/xrEngine/xr_input.cpp [hunk 1]
```diff
@@ -236,6 +236,7 @@ bool CInput::iGetAsyncBtnState(int btn)
 void CInput::GrabInput(const bool grab)
 {
     // Self descriptive
+
```
Comment (Xottab-DUTY) [inline src/xrEngine/xr_input.cpp:239]:
```suggestion
```

File: src/xrEngine/main.cpp [hunk 2]
```diff
@@ -209,12 +209,12 @@ ENGINE_API void Startup()
...
     if (!g_bBenchmark && !g_SASH.IsRunning())
 #endif
-        destroySettings();
+    destroySettings();
     LALib.OnDestroy();
 #if !defined(LINUX)
     if (!g_bBenchmark && !g_SASH.IsRunning())
 #endif
-        destroyConsole();
+    destroyConsole();
```
Comment (Xottab-DUTY) [inline src/xrEngine/main.cpp:217]:
```suggestion
        destroyConsole();
```

File: src/xrEngine/main.cpp [hunk 3]
```diff
@@ -209,12 +209,12 @@ ENGINE_API void Startup()
 #if !defined(LINUX)
     if (!g_bBenchmark && !g_SASH.IsRunning())
 #endif
-        destroySettings();
+    destroySettings();
```
Comment (Xottab-DUTY) [inline src/xrEngine/main.cpp]:
```suggestion
        destroySettings();
```

File: src/xrEngine/Device_create.cpp [hunk 4]
```diff
@@ -94,6 +94,7 @@ void CRenderDevice::UpdateWindowProps(const bool windowed)
         SDL_SetWindowFullscreen(m_sdlWnd, maximalResolution ? SDL_WINDOW_FULLSCREEN_DESKTOP : 0);
         SDL_SetWindowSize(m_sdlWnd, psCurrentVidMode[0], psCurrentVidMode[1]);
         SDL_SetWindowBordered(m_sdlWnd, drawBorders ? SDL_TRUE : SDL_FALSE);
+
```
Comment (Xottab-DUTY) [inline src/xrEngine/Device_create.cpp]:
```suggestion
```

File: src/xrEngine/Device_create.cpp [hunk 5]
```diff
@@ -307,3 +308,4 @@ void FreeRefreshRateToken()
 
     VidRefreshRateToken.clear();
 }
+
```
Comment (Xottab-DUTY) [inline src/xrEngine/Device_create.cpp]:
```suggestion
```

File: src/xrCore/_std_extensions.h [hunk 6]
```diff
@@ -265,6 +275,7 @@ inline void MemFill32(void* dst, u32 value, size_t dstSize)
         *ptr++ = value;
 }
 
+
```
Comment (Xottab-DUTY) [inline src/xrCore/_std_extensions.h]:
```suggestion
```

File: src/xrCore/Threading/Lock.cpp [hunk 7]
```diff
@@ -15,7 +15,6 @@ struct LockImpl
     ICF bool TryLock() { return !!TryEnterCriticalSection(&cs); }
 #else
     std::recursive_mutex mutex;
-
```
Comment (Xottab-DUTY) [inline src/xrCore/Threading/Lock.cpp]: Please, return this deleted line.

File: src/xrAICore/Components/script_world_property_script.cpp [hunk 8]
```diff
@@ -20,4 +20,4 @@ SCRIPT_EXPORT(CScriptWorldProperty, (), {
                          .def("value", &CScriptWorldProperty::value)
                          .def(const_self < other<CScriptWorldProperty>())
                          .def(const_self == other<CScriptWorldProperty>())];
-});
+})
```
Comment (Xottab-DUTY) [inline src/xrAICore/Components/script_world_property_script.cpp]:
```suggestion
});
```

**Review by Xottab-DUTY — CHANGES_REQUESTED** [review]

File: src/xrAICore/Components/script_world_property_script.cpp (hunk 8 above)
Comment (xrombik) [reply src/xrAICore/Components/script_world_property_script.cpp]: Extra ';` - why?

Comment (Xottab-DUTY) [reply src/xrAICore/Components/script_world_property_script.cpp]: This PR is about bugfixes, not cosmetic changes.

> Котлеты отдельно, мухи отдельно. ©️ Путин

**Pull request closed without merging**